packages/@aws-cdk/custom-resources/lib/provider-framework/runtime/cfn-response.ts

```diff
@@ -24,7 +27,7 @@ export async function submitResponse(status: 'SUCCESS' | 'FAILED', event: CloudF
     Reason: options.reason || status,
     StackId: event.StackId,
     RequestId: event.RequestId,
-    PhysicalResourceId: event.PhysicalResourceId || event.RequestId,
+    PhysicalResourceId: event.PhysicalResourceId as any,
```

Comment:
Is it infeasible to type this more strictly?

Reply:
I am using [@types/aws-lambda](https://www.npmjs.com/package/@types/aws-lambda) for types and it makes it a bit difficult.

Reply:
okay, moved the fallback marker to here so `as any` is gone. Thanks for keeping me honest :-)